controller/workitem_blackbox_test.go

```diff
@@ -1982,6 +1982,27 @@ func (s *WorkItem2Suite) TestCreateWorkItemWithInvalidSpace() {
 	test.CreateWorkitemBadRequest(t, s.svc.Context, s.svc, s.wi2Ctrl, &c)
 }
 
+// Following test fails because Iterations table is being cleaned up
+// and default iteration for root space is gone
+// func (s *WorkItem2Suite) TestDefaultSpaceAndIterationRelations() {
+// 	t := s.T()
+// 	c := minimumRequiredCreateWithType(workitem.SystemFeature)
+// 	title := "Solution on global warming"
+// 	c.Data.Attributes[workitem.SystemTitle] = title
+// 	c.Data.Attributes[workitem.SystemState] = workitem.SystemStateNew
+// 	_, wi := test.CreateWorkitemCreated(t, s.svc.Context, s.svc, s.wi2Ctrl, &c)
+// 	require.NotNil(t, wi)
+// 	require.NotNil(t, wi.Data.Relationships)
+// 	require.NotNil(t, wi.Data.Relationships.Iteration)
+
+// 	spaceRepo := space.NewRepository(s.db)
+// 	spaceInstance, err := spaceRepo.Load(s.svc.Context, space.SystemSpace)
+// 	iterationRepo := iteration.NewIterationRepository(s.db)
+// 	defaultIteration, err := iterationRepo.LoadDefault(context.Background(), *spaceInstance)
+// 	require.Nil(t, err)
+// 	assert.Equal(t, defaultIteration.ID.String(), *wi.Data.Relationships.Iteration.Data.ID)
+// }
+
```

Comment:
@sbose78 back to square one !
I am wondering [this change](https://github.com/almighty/almighty-core/pull/996/files#diff-483924d5da75e87a58002b6e3a035ea9R167) is not sufficient. Because I could see that default area and default iteration for root space are being cleaned up at that point.